public_dropin_gpu_environments/vllm/dr_requirements.in

```diff
@@ -1,5 +1,6 @@
-openai
 datarobot
 datarobot-mlops
 datarobot-mlops-connected-client
 datarobot-drum
+openai
```

Comment:
I just found that I have Triton env broken (it fails to `import openai`), becase the `BaseGpuPredictor` defined in the `__init__.py`.

Reply:
Oh shoot, that's my fault. I'll move the base class out of `__init__.py` because we shouldn't pull in third party deps there then.